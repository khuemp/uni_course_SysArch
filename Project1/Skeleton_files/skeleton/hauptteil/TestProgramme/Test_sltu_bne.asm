addiu $1, $0, 5
addiu $2, $0, 7
sltu $3, $1, $2
bne $3, $0, target
addiu $4, $0, 23
j end
target:
addiu $4, $0, 42
end: