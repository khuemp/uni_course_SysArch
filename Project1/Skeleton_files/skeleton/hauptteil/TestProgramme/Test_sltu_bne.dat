0x24010005
0x24020007
0x0022182b
0x14600002
0x24040017
0x08100007
0x2404002a
