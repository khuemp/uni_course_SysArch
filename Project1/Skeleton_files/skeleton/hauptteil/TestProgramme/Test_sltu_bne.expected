@1 5
@2 7
@3 1
@4 2a
